src/docfx/build/context/Output.cs

```diff
@@ -84,7 +84,7 @@ public void WaitForCompletion()
             _queue.Completion.Wait();
         }
 
-        private string GetDestinationPath(string destRelativePath)
+        public string GetDestinationPath(string destRelativePath)
```

Comment:
Rather than make this method public, prefer a `WriteLines` method to:
1. Allow encapsulation of async write operations using a private queue
2. Restrict file operations inside `Input` and `Output` class so we can swap the implementation without caller awareness.

Reply:
sounds good